### PR TITLE
Cookie bar styling

### PR DESCRIPTION
**Pull request by helenb**
[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1367264493)

### Description of Changes Made

There is no design for this - I've just applied some fairly basic styles

### How to Test

- Delete any cookies for localhost (under 'Applications' in the chrome dev tools)
- Refresh the page to see the cookie banner.
- Choose to accept or reject cookies in order to see the cookie banner dismissed
- Reload the page to ensure the cookie banner no longer appears

Points to note
- The link to 'find out more' is hard-coded - I will update the ticket to alert Anna to this - this was the same on the previous site
- I have asked whether we should be using a new google analytics snippet or updating the existing one - currently we are using the same one as on the previous site, but analytics cookies do not get set even if you accept cookies - I suspect because the google analytics set-up is not configured to work with the localhost domain or the new staging site domain.

### Screenshots

<details>
  <summary>Expand to see more</summary>

Screenshot 2024-02-19 at 12 49 02

</details>

### MR Checklist

- [x] Add a description of your pull request and instructions for the reviewer to verify your work.
- [x] If your pull request is for a specific ticket, link to it in the description.
- [x] Stay on point and keep it small so the merge request can be easily reviewed.
- [x] Tests and linting passes.

#### Unit tests

- [ ] Added
- [x] Not required

#### Documentation

- [ ] Updated build docs
- [ ] Updated editor guidelines (https://docs.google.com/document/d/1PAWccdQ4tfaZsrEWmpDhvP3GH5RRmBOARFVp4b-kje8/edit?usp=sharing)
- [x] Not required

#### Browser testing

- [x] I have tested in the following browsers and environments (edit the list as required)
  - Latest version of Chrome on mac
  - Latest version of Firefox on mac
  - Latest version of Safari on mac
  - Safari on last two versions of iOS
  - Chrome on last two versions of Android
- [ ] Not required

#### Data protection

- [x] Not relevant
- [ ] This adds new sources of PII and documents it and modifies Birdbath processors accordingly

#### Accessibility

- [x] Automated WCAG 2.1 tests pass
- [x] Manual WCAG 2.1 tests completed
- [x] I have tested in a screen reader
- [x] I have tested in high-contrast mode
- [x] Any animations removed for prefers-reduced-motion
- [ ] Not required

#### Sustainability

- [ ] Images are optimised and lazy-loading used where appropriate
- [ ] SVGs have been optimised
- [ ] Perfomance and transfer of data considered
- [ ] If JavaScript is needed alternatives have been considered
- [x] Not required

#### Pattern library

- [ ] The pattern library component for this template displays correctly, and does not break parent templates
- [x] The styleguide is updated if relevant
- [ ] Changes are not relevant the pattern library

### CODE REVIEW

**Comment by chris-lawton** [comment]
Does it need adding to the base page template? Looks great though, nice one.

**Comment by helenb** [comment]
> Does it need adding to the base page template? Looks great though, nice one.

It's included in the footer template, which seems to work ok.

**Comment by chris-lawton** [comment]
> > Does it need adding to the base page template? Looks great though, nice one.
> 
> It's included in the footer template, which seems to work ok.

Ah ok, I couldn't see it on the home page in incognito, not sure why.

File: tbx/project_styleguide/templates/patterns/molecules/cookie-message/cookie-message.html [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+<div data-cookie-message class="cookie-message {{ classes }}">
+    <div class="cookie-message__container grid">
+        <div class="cookie-message__message grid__cookie-message">
+            <p>We use cookies to measure how you use the website. We want our site to be easy for you to use; understanding how you interact with it helps us know that. <a aria-label="Find out more about how we use cookies" href="/cookies/" class="cookie-message__link">Find out more</a>.</p>
         </div>
-        <div>
-            <a id="opt-in-torchbox-cookie" href="#" data-cookie-opt-in>Opt in</a>
-            <a id="opt-out-torchbox-cookie" href="#" data-cookie-opt-out>Opt out</a>
+        <div class="cookie-message__action grid__cookie-message">
+            <a class="button" id="opt-in-torchbox-cookie" href="#" data-cookie-opt-in>Opt in</a>
```
[the remark this answers is not in the document]
Comment (helenb) [reply tbx/project_styleguide/templates/patterns/molecules/cookie-message/cookie-message.html]: The perils of re-using old code :)  https://github.com/torchbox/wagtail-torchbox/commit/d9000c18d5ff7ac5280694bfee79e83f1066d5c3

(I wasn't looking through the old code to catch you out - I was tracking down if we need the ids, and it turns out we do).

**Pull request closed without merging**